### DISCUSSION TITLE
RAM-Hour measurment

### DISCUSSION THREAD

**Discussion by ZahirBilal** [online-ml/river#1019] [Q&A] [answered]
Hello Everyone,

Is there a way to evaluate an algorithm (prequentially) and measure the RAM-Hour consumed during the evaluation instead of MBs? 

"Every GB of RAM deployed for 1 hour equals one RAM-Hour." - https://informatica.si/index.php/informatica/article/viewFile/428/432

Best wishes,
Zahir

**Comment by ZahirBilal** [comment]
PS: I can somehow convert the memory consumed in MBs to RAM-Hour. Nevertheless, I wanted to know if I can get the value directly, that would be easier :) 

Thanks again

**Comment by MaxHalford** [comment, marked as answer]
Hey @ZahirBilal, apologies for the late reply. The answer is that no, this information isn't directly available. It could be cool to add it though. Note that you can use the `iter_progressive_val_score` function to access the amount of RAM in use at different checkpoints.

**Reply by ZahirBilal** [reply]
Thanks for the answer Max. That is exactly what I am looking to transform to RAH HOUR . I will work on it later to find the exact transformation equation, then will see how it fits in River ;)
